README.rst

```diff
@@ -113,50 +113,32 @@ can then be transferred to the XenServer dom0 and run natively.
 
 KVM support
 ~~~~~~~~~~~
-LibVMI will have KVM support if libvirt is available during compile time. Ensure that your
-libvirt installation supports QMP commands, most prepackaged versions do not support this
-by default so you may need to install libvirt from source yourself.  To enable QMP support
-when installing from source, ensure that you have libyajl-dev (or the equivalent from your
-linux distro) installed, then run the configure script from libvirt.  Ensure that the
-configure script reports that it found yajl.  Then run make && make install.
+The KVM driver for LibVMI is based on the new ``KVMi`` subsystem for
+KVM, which aims to bring an official VMI API on this hypervisor.
 
-Currently there is no native VMI support in KVM, so you have two options for adding it:
+The current state of these KVM patches is a work-in-progress, but it is mature
+enough to be proposed by default in LibVMI, instead of using previous memory access
+methods (GDB stub or the ``tools/qemu-kvm-patch`` memaccess custom patches for QEMU).
 
-  1) Patch QEMU-KVM with the provided patch.  This technique will
-     provide the fastest memory access, but is buggy and may cause
-     your VM to crash / lose data / etc.  To use this method,
-     follow the instructions in the libvmi/tools/qemu-kvm-patch
-     directory.
+In order to use the LibVMI KVM driver, you will have to setup ``KVM-VMI`` beforehand.
+KVM-VMI is a Github community dedicated to bring a native VMI API on KVM, and
+currently hosting this new API:
 
-  2) Enable GDB access to your KVM VM.  This is done by adding
-     '-s' to the VM creation line or, by modifying the VM XML
-     definition used by libvirt as follows:
+https://github.com/KVM-VMI/kvm-vmi
 
-     - Change:
+Follow this wiki page to setup KVM-VMI:
 
-       .. code::
+https://github.com/KVM-VMI/kvm-vmi/wiki/KVM-VMI-setup
```

Comment:
@tklengyel I updated the `README` already in my PR, you can review the instructions.

Regarding the Travis, I will get it fixed today.